Services/Init/classes/class.ilInitialisation.php

```diff
@@ -1671,7 +1671,15 @@ protected static function redirect($a_target, $a_message_id = '', array $a_messa
             !stristr($a_target, ILIAS_HTTP_PATH)) {
             $a_target = ILIAS_HTTP_PATH . "/" . $a_target;
         }
-        
+
+        foreach (['ext_uid', 'soap_pw'] as $param) {
+            if (false === strpos($a_target, $param . '=') && isset($GLOBALS['DIC']->http()->request()->getQueryParams()[$param])) {
```

Comment:
isset on method calls is not suported.

Reply:
Correct. But fails with fatal: PHP Fatal error:  Cannot use isset() on the result of a function call (you can use "null !== func()" instead) in 5.6